example/cra-provider/src/connectors/walletConnect.ts

```diff
@@ -0,0 +1,26 @@
+import { initializeConnector } from '@web3-react/core'
+import { WalletConnect } from '@web3-react/walletconnect'
+
+export const [walletConnect, walletConnectHooks] = initializeConnector<WalletConnect>((actions) => {
+  const mainnetKey = process.env.REACT_APP_ALCHEMY_KEY_MAINNET
```

Comment:
why don't just use getEnv from @arcxmoney/core

Reply:
I don't want to add a package with all these deps just to use `getEnv`: 
```
  "dependencies": {
    "@arcxgame/contracts": "^11.4.0",
    "@google-cloud/storage": "^5.16.1",
    "@restless/ethereum": "^0.3.0",
    "@restless/restless": "^0.4.2",
    "@restless/sanitizers": "^0.2.5",
    "axios": "^0.21.1",
    "dayjs": "^1.10.6",
    "dotenv": "^8.2.0",
    "ethers": "5.5.4",
    "ethers-multicall": "^0.2.3",
    "lodash": "^4.17.21",
    "mongodb": "^4.2.0",
    "node-cache": "^5.1.2",
    "rollbar": "^2.24.0"
  },
```

Reply:
Just create one then :) With getting env throwing if doesn't exist. Useful utility

Reply:
As discussed, won't create it bc it's just used here